libraries/system.rb

```diff
@@ -26,7 +26,7 @@ def initialize(virtualization_systems)
       end
 
       def vm?
-        @virtualization_systems.empty? || @virtualization_systems.values.include?('guest') ? true : false
+        @virtualization_systems.empty? || @virtualization_systems.value?('guest') ? true : false
```

Comment:
I thought the conditional already returned a bool?

Reply:
They're your cookstyle changes ;-)